docs/en/observability/synthetics-quickstart-fleet.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[synthetics-quickstart-fleet-setup]]
+== Set up Fleet Server
+
+Synthetic monitoring via {agent} and {fleet} requires {fleet-server} to be set up. If you are using Elastic Cloud fleet server will already be provided and you can skip this step. To learn more, see {fleet-guide}/fleet-quick-start.html#set-up-fleet[{fleet}].
+
+[discrete]
+[[synthetics-quickstart-fleet-agent-policy]]
+== Create an agent policy for Elastic Synthetics
+
+Use a separate fleet policy for Synthetics monitors in most cases. It is generally the case that synthetics monitors should run on nodes dedicated to that purpose. See {fleet-guide}/agent-policy.html#create-a-policy[{agent} policy] for more information on Fleet policies and creating them.
+
+[role="screenshot"]
+image::images/synthetics-agent-policy.png[Synthetics integration]
+
+[discrete]
+[[synthetics-quickstart-fleet-add-container]]
+== Add an {agent} to {fleet}, using the special *elastic-agent-complete* container
+
+{agent} is a single, unified agent that you can deploy to hosts or containers to collect data and send it to the {stack}. Behind the scenes, {agent} runs {beats} or {elastic-endpoint} required for your configuration. 
+
+To run synthetics tests, you must use the *elastic-agent-complete* Docker image to create a self-hosted {agent} node. This image contains the dependencies to run synthetic monitors. The standard hosted or self-hosted {agent} will not work.
+
+To add an {agent} to {fleet}, you'll need an enrollment token and the URL of your fleet server. You can use the default enrollment token for your policy or create new enrollment tokens as needed. To learn more, see {fleet-guide}/elastic-agent-container.html[Run Elastic {agent} in a container] and {fleet-guide}/fleet-enrollment-tokens.html[{fleet} enrollment tokens] for more information. Please note that you must run the docker container as a non-root user for Browser monitors. See <<synthetics-quickstart-fleet-security>>  for more information on security requirements and compatibility within the container.
+
+You may need to set additional environment variables. See the {fleet-guide}/agent-environment-variables.html[{agent} environment variables guide] for more information. See {fleet-guide}/elastic-agent-container.html[Run {agent} in a container] for more information on running {agent} with Docker.
+
+[[synthetics-quickstart-fleet-configure-policy]]
+== Configure an Elastic Synthetics integration policy beta[]
+
+Elastic Synthetics tests can be configured through {fleet} using the Elastic Synthetics integration. To learn more, see {fleet-guide}/fleet-quick-start.html#add-synthetics-integration[Elastic Synthetics].
+
+[role="screenshot"]
+image::images/synthetics-integration.png[Synthetics integration]
+
+In the example above, our library of synthetic tests is downloaded from the
+remote zip endpoint for https://github.com/elastic/synthetics-demo/tree/main/todos/synthetics-tests[our todos example]. 
+The fields should be set as follows:
```

Comment:
Should this be a list? If so, it's rendering incorrectly. Asciidoc lists require a blank line between previous content and the start of the list.

<img width="747" alt="Screen Shot 2021-09-16 at 2 05 02 PM" src="https://user-images.githubusercontent.com/5618806/133685484-1391418d-2b2d-47d4-80aa-55163352434f.png">

This should fix it:

```suggestion
The fields should be set as follows:

```